jenkins/scripts/bare_metal_lab/files/config.sh

```diff
@@ -1,168 +0,0 @@
-#!/bin/bash
-
-#
-# Choose whether the "baremetal" libvirt network will use IPv4, IPv6, or IPv4+IPv6.
-# This network is the primary network interface for the virtual bare metal hosts.
-#
-# Note that this only sets up the underlying network, and fully provisioning IPv6
-# kubernetes clusters is not yet automated.  If IPv6 is enabled, DHCPv6 will
-# be available to the virtual bare metal hosts.
-#
-# v4   -- IPv4 (default)
-# v6   -- IPv6
```

Comment:
I think this file was there as for the reference of a doc to explain to user of BMH shortly what each variable is for. Do we really want to drop it?

Reply:
I thought this was just a duplicate of [the one we have in metal3-dev-env](https://github.com/metal3-io/metal3-dev-env/blob/master/config_example.sh). Do we need both?

Reply:
Ah I see, didn't notice it. Yeah, I would be +1 for dropping it, not sure why we added